tests/upgrade-service.spec.ts

```diff
@@ -43,17 +43,21 @@ describe('Upgrade Service', () => {
     const result = await upgradeService.upgradeDeployment();
     const resultAfter = await upgradeService.getCurrentVersion('busybox');
 
-    expect(resultBefore).to.contain('1.34');
-    expect(resultAfter).to.contain('1.35');
+    expect(resultBefore).to.equal('busybox:1.34 busybox:1.34');
+    expect(resultAfter).to.contain('busybox:1.35 busybox:1.35');
 
     expect(result).to.deep.equal({
       upgradeResult: 1,
-      upgradedContainers: { busybox: { ok: true } },
+      upgradedContainers: {
+        busybox: { ok: true },
+        'busybox-1': { ok: true },
+      },
     });
   });
 
   it('Should upgrade deployment if container is named with a suffix', async () => {
-    const upgradeMessageArray: IUpgradeMessage[] = [{ container_name: 'busybox', image_tag: 'busybox:1.35' }];
+    await runCommand(`sleep 5`, 'Avoid 409 conflict race condition with previous test');
```

Comment:
Minor: Is there a reason to use `runCommand` with sleep rather than `setTimeout` as you do above? I see it's being used already in this file and since it's only being used with already expensive tests so the minor overhead doesn't matter but just curious

Reply:
I was also being lazy about resolving the race condition itself. I think reusing the same deployments in consecutive tests is pretty bad, because a failure can cascade to other tests, so the correct fix would be to use different containers for every test.